docs/content/en/docs/tutorials/_index.md

```diff
@@ -22,10 +23,3 @@ When deploying to a remote cluster you have to point Skaffold to your default im
 ### What's next
```

Comment:
It's odd seeing a `what's next` before the actual content.  TBH, I think we should:

1. turn the default-repo into a callout `{{ <alert> }}` — but really this should be incorporated into each of the tutorial pages
2. move the survey into the footer shown on each page
2. get rid of most of the remaining text

Reply:
ya, those definitely seem out of place

Reply:
how about now?
http://34.94.56.85:1313/docs/tutorials/

moved Survey to footer and named it `Feedback`
<img width="406" alt="image" src="https://user-images.githubusercontent.com/39389231/131633317-8099cfbb-2f73-46bc-af1f-92b9c1f8f897.png">